code/controllers/subsystem/movement/movement_types.dm

```diff
@@ -391,17 +403,25 @@
 	src.simulated_only = simulated_only
 	src.avoid = avoid
 	src.skip_first = skip_first
+	src.initial_path = initial_path
```

Comment:
you should make this copy, otherwise it will modify anything else that uses this list once it's passed in, and we don't really want that

Reply:
good point :) will fix